### DISCUSSION TITLE
Is it possible to modify the default serialize.blockedlist in Dubbo? I want to allow exceptions from the org.springframework package to be passed through and return the original exception to the caller. However, by default, serialization of classes under the org.springframework package is not allowed.

### DISCUSSION THREAD

**Discussion by lichangan18356288293** [apache/dubbo#14703] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

I am using Dubbo version 3.2.7. What are some ways to achieve this?

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
You can add to serialize.allowlist
